sdk/helper/certutil/types.go

```diff
@@ -525,6 +539,12 @@ func (p *ParsedCSRBundle) getSigner() (crypto.Signer, error) {
 			return nil, errutil.UserError{Err: fmt.Sprintf("Unable to parse CA's private RSA key: %s", err)}
 		}
 
+	// case Ed25519PrivateKey:
```

Comment:
Can the commented code be removed?

Reply:
i need to add this back but it was throwing a compile error and left it as a todo. Let me verify what's wrong since I am pretty sure i need to implement it if I want vault to sign ed25519 CSR it receives.